integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -104,9 +104,10 @@ const getSauceLabsConfig = (capabilities) => {
     tags: [capabilities.name],
     seleniumVersion: '3.141.59',
     tunnelIdentifier: process.env.JOB_ID,
-    ...(capabilities.platform.toUpperCase() !== 'LINUX' && {
-      extendedDebugging: true
-    })
+    ...(capabilities.platform.toUpperCase() !== 'LINUX' && 
+      capabilities.extendedDebugging != false && {
```

Comment:
Checking if this can be kept as earlier. I think this is the first and starting place where we set the extendedDebugging to true so I am not sure if we have `capabilities.extendedDebugging` as valid. You can keep this as is I guess and the change in SDKBaseTest should be fine to override right?

Reply:
Request testing it once locally.